tests/e2e_tests

```diff
@@ -36,14 +37,33 @@ do
         echo "" >> ../error.log
     fi
     cd ..
```

Comment:
maybe it would be better to capture the current working dir before the cd and restore it, that way the test could make use of a cd without breaking this script:

```suggestion
pwd=$(pwd)
cd $dir

# ...

cd $pwd
```

Reply:
actually, our tests don't do `cd` and I pretty sure they shouldn't. Left as is for now, but probably I miss something